src/components/network-map-tab.js

```diff
@@ -848,15 +848,17 @@ export const NetworkMapTab = ({
             {mapEquipments?.substations?.length > 0 &&
                 renderNominalVoltageFilter()}
 
-            {displayOverloadTable && isLoadFlowValid() && linesNearOverload() && (
-                <div className={classes.divOverloadedLineView}>
-                    <OverloadedLinesView
-                        lineFlowAlertThreshold={lineFlowAlertThreshold}
-                        mapEquipments={mapEquipments}
-                        disabled={disabled}
-                    />
-                </div>
-            )}
+            {displayOverloadTable &&
```

Comment:
Something strange here ? whitespaces ?

Reply:
It's linked to a prettier update I reckon